### ISSUE TITLE
Intune Integration - Make documentation more prescriptive

### ISSUE THREAD

**Issue by steviecoaster** [chocolatey/docs#346]
Currently the Intune Documentation on the website is split into the following sections:

- Prereqs
- Intune Configuration
- Converting packages
- Pushing packages

This can be _slightly_ hard to follow when first setting up things to get started using the Intune Integration. I propose the following sections

- Prereqs (You need _these_ things to use this functionality)
- Preparing your Intune Tenant (do these things inside the Azure portal)
- Preparing your workstation (Run these commands to establish your configuration)
- Converting a Chocolatey Package for Intune (self-explanatory)
- Pushing a Chocolatey Package for Intune (self-explanatory)
- Video Tutorial (self-explanatory)
- Troubleshooting (self-explanatory)

**Comment by ryanrichter94** [comment]
Suggestion. Might re-word the Preparing your workstation part to Configuring Chocolatey.
